server/models/queries/mailList.js

```diff
@@ -0,0 +1,20 @@
+const base = require('../config');
+
+exports.addMail = email => {
+  return base('mailing_list').create([{ fields: { email } }]);
+};
+
+exports.getEmails = async () => {
+  let records = [];
+
+  const processPage = (partialRecords, fetchNextPage) => {
+    records = [...records, ...partialRecords.map(r => r.get('email'))];
+    fetchNextPage();
+  };
+
+  await base('mailing_list')
+    .select({ view: 'Grid view', pageSize: 2 })
```

Comment:
why did you specify a page size of 2? 
```suggestion
    .select({ view: 'Grid view' })
```